### PR TITLE
release as version 1.2.1

### PR DESCRIPTION
**Pull request by adon-at-work**
- added ya and yublf private filters
- updated frCoPt() to properly handle null and undefined

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "xss-filters",
-  "version": "1.1.2",
+  "version": "1.2.1",
```
Comment (neraliu) [inline bower.json]: just curios that is the version 1.2.0 live in bower?

Comment (adon-at-work) [inline bower.json]: yes, since git tag is there. but will give a warning

Comment (neraliu) [inline bower.json]: alright

**Pull request merged**